tests/features/secure-files.feature

```diff
@@ -71,12 +103,17 @@ Feature: VermillhFeature: Vermillion is able to handle secure files
                 When The consumer downloads file by passing an empty resource id
 		        Then The response status should be 400
 
-	Scenario: Download secure-file with only token
-                Given Vermillion is running
-                When The consumer downloads file by passing only token
-                Then The response status should be 403
+        Scenario: Download file with public id and token
+          Given Vermillion is running
+          When The consumer downloads file by passing public id and token
+          Then The response status should be 400
```

Comment:
If a public file is downloaded with a token, the status should not be 400. Could you describe what is happening here?

Reply:
Only secure files can be downloaded here..

if (resourceIDStr.endsWith(".public")) {
                apiFailure(
                        context,
                        new BadRequestThrowable(
                                "This API is for secure resources only. Use /provider/public endpoint to explore public data"));
                return;
            }
        }